lib/Cake/Test/Case/Utility/CakeFinalizerTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * CakeFinalizerTest
+ *
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @package       Cake.Utility
+ * @since         CakePHP(tm) v 2.10.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+App::uses('CakeFinalizer', 'Utility');
+
+/**
+ * CakeFinalizerTest class
+ *
+ * @package       Cake.Test.Case.View.Helper
+ */
+class CakeFinalizerTest extends CakeTestCase {
+
+/**
+ * Tests that the given callable function is called by the CakeFinalizer
+ *
+ * @return void
+ */
+	public function testCallable() {
+		$mock = $this->getMock('stdClass', array('finalize'));
```

Comment:
We should probably start using the un-deprecated `getMockBuilder()` method for new tests.